book/src/frequently_asked_questions.md

```diff
@@ -78,6 +94,12 @@ mitigations, authorisation code interception is not prevented by the presence or
 We would strongly encourage OAuth2 client implementations to implement and support PKCE, as it
 provides defense in depth to known and exploited authorisation code interception attacks.
 
+### Why do you allow disabling PKCE but not TLS?
+
+Because there are still many applications where PKCE is not available and it is not trivial to solve
```

Comment:
I think I'd rather focus on "in the case that PKCE is absent on a single oauth2 client, the scope of failure is reduced to that single client. In the case of compromise of an internal network between a load balancer and Kanidm, the attacker can access and steal all tokens we ever issue. "